### PR TITLE
This is the simple merge

### PR DESCRIPTION
**Pull request by bjmckenz**
In support of #93, I had to move a few things around and change a few template variable names. That, plus fixing links, got it up to 50 files. Very sorry, I was trying to keep it super lean.

Purpose is that this is the original frames/windows, but with all the prep work done so that the window stuff is a cleaner subsequent merge.

I'm going to wait til this is...addressed...before proceeding. I have the other changes stashed but I don't want to proceed until this is reasonably clean.

Testing?
* no change of func
* I've clicked through everything and noted a few issues. But otherwise should be OK.
*

### CODE REVIEW

File: student-info-system/schooladmin/views/__init__.py [hunk 1]
```diff
@@ -499,12 +609,18 @@ def section_new_helper(request, semester_id=None, courseid=None):
         })
 
 
-@role_login_required(Profile.ACCESS_ADMIN, Profile.ACCESS_PROFESSOR)
+@login_required
```
Comment (oceancypress) [inline student-info-system/schooladmin/views/__init__.py:612]: ?

File: student-info-system/student/views.py [hunk 2]
```diff
@@ -292,7 +292,7 @@ def request_major_change(request):
 
     the_major = the_user.profile.student.major
     data = {
-        'user': the_user,
+        'auser': the_user,
```
Comment (oceancypress) [inline student-info-system/student/views.py:295]: how come user got switched to auser everywhere?

**Comment by oceancypress** [comment]
> Testing?
> 
> * no change of func
> * I've clicked through everything and noted a few issues. But otherwise should be OK.


Were the issues issues before this or became issues in this branch?

File: student-info-system/schooladmin/views/__init__.py (hunk 1 above)
Comment (bjmckenz) [reply student-info-system/schooladmin/views/__init__.py:612]: all three roles use it.

File: student-info-system/student/views.py (hunk 2 above)
Comment (bjmckenz) [reply student-info-system/student/views.py:295]: in prep for frameless. *It turns out* that the logged-in user is passed in the context as "user", so if you want to do something with it, you best not pass in values with the same name. I'm using it to determine which home screen to show (in the uncommitted work).

That was the main drive for this branch actually -- it should be functionally neutral but is testable.

Comment (oceancypress) [reply student-info-system/student/views.py:295]: why don't we use the user that is automatically passed in?

**Comment by oceancypress** [comment]
tested and look good, just waiting on one question

**Review by oceancypress — APPROVED** [review]

**Pull request merged**